test/editions/SingleBatchEditionTest.sol

```diff
@@ -125,8 +130,17 @@ contract SingleBatchEditionTest is Test, ShowtimeVerifierFixture {
     //////////////////////////////////////////////////////////////*/
 
     function testCreateEditionHappyPath() public {
+        uint256 id = uint256(keccak256(abi.encodePacked(creator, "name", "animationUrl", "imageUrl")));
+        address expectedAddr = address(editionCreator.getEditionAtId(id));
```

Comment:
yo @jorgeavaldez check this out! With this version, we can predict the address of the future edition ahead of time